src/shared/model/Routes.ts

```diff
@@ -42,6 +42,7 @@ export const ValidRoutePathsArray = [
   '/welcome/email-sent',
   '/welcome/expired',
   '/welcome/resend',
+  '/agree/GRS',
```

Comment:
Does this need to be changed in identity-platform too, i,e point it to gateway instead of identity-frontend?

Reply:
ah yes — I had it ready in a branch — raised the PR here: https://github.com/guardian/identity-platform/pull/529